src/Core/src/Layouts/LayoutExtensions.cs

```diff
@@ -43,5 +43,13 @@ public static Rectangle ComputeFrame(this IFrameworkElement frameworkElement, Re
 			return new Rectangle(bounds.X + margin.Left, bounds.Y + margin.Top,
 				frameWidth, frameHeight);
 		}
+
+		static Thickness GetMargin(this IFrameworkElement frameworkElement)
```

Comment:
This is only needed because the framework element no longer has margin, but I got a comment on maybe we do want it still.

Reply:
i'll just revert the margin changes for now to limit the number of things addressed in this PR

Reply:
Actually, I'm going to just leave it like this for now. We currently don't have margin on VisualElement so we have to add extra code on Visualelement to now account for a margin that doesn't exist. Putting Margin on IView keeps it inline with our current structures and I think we can modify once we organize it all a bit better